stacks/BasicApiStack.ts

```diff
@@ -118,19 +118,22 @@ export function BasicApiStack ({ app, stack }: StackContext): { queue: Queue, bu
       'GET    /internal/pins/{cid}': {
         function: {
           handler: 'basic/get-pin.handler',
-          functionName: formatResourceName(app.stage, 'getPin')
+          functionName: formatResourceName(app.stage, 'getPin'),
```

Comment:
Why do we need a timeout for the internal lambda as well? Do we have any timeout on them?